### DISCUSSION TITLE
Fiz um aplicativo com a API Spotify no meu primeiro aplicativo com React Native

### DISCUSSION THREAD

**Discussion by belmirofss** [frontendbr/forum#2089] [Mostre seu trabalho]
## Fiz um aplicativo com a API Spotify no meu primeiro aplicativo com React Native

Eu praticamente nasci e fui criado no Angular, ou seja, experiência totalmente voltada ao *framework* mantido pela Google. Resolvi sair um pouco da zona de conforto e aprender React de fato.

Minha decisão para aprender, foi construir um aplicativo que não fosse inútil e tivesse o mínimo de valor. [O resultado foi um aplicativo consumindo a API do Spotify](https://github.com/belmirofss/statsfy) e um [artigo no Medium.](https://medium.com/reactbrasil/minha-primeira-experi%C3%AAncia-com-o-react-native-b972722a2e67)

**Comment by lucalves** [comment]
Parabéns. Acabei de deixar minha estrela lá no repositório. 😉
